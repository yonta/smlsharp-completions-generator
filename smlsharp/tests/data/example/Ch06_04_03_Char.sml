#"a";
#"\n";

ord #"a";
chr (97 + 7);
ord #"a" - ord #"A";
chr (ord #"H" + 32);
