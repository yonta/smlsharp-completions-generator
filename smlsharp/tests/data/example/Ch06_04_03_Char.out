val it = #"a" : char

val it = #"\n" : char

val it = 97 : int

val it = #"h" : char

val it = 32 : int

val it = #"h" : char
